spec/forms/csv_upload_form_spec.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CsvUploadForm, type: :form do
+  describe '#save' do
+    context 'given a valid file of 10 keywords' do
+      it 'can save' do
+        user = Fabricate(:user)
+        form = described_class.new(user)
+        saved = form.save(file_fixture('csv/valid.csv'))
+        expect(saved).to be(true)
+      end
+
+      it 'saves 10 keywords with the correct user' do
+        user = Fabricate(:user)
+        form = described_class.new(user)
+        form.save(file_fixture('csv/valid.csv'))
+        expect(keyword_for_user(user).count).to eq(10)
```

Comment:
Usually when we try to assert the count change or not change after an insert, we can use this syntax in Rspec. 

```suggestion
expect do
  form.save(file_fixture('csv/valid.csv'))
end.to change(Keyword, :count).by(10)
```

Reply:
I think the assertion here is different, where `change(Keyword, :count).by(10)` checks for the 10 addition but `expect(keyword_for_user(user).count).to eq(10)` checks if the  keywords belongs to the correct user.

Reply:
Ah I got your point here now. Thanks!